### PR TITLE
fix(linter): Fix the linter configuration

### PR DESCRIPTION
**Pull request by xmnlab**
## Pull Request description

Fix the linter configuration for the project and the template.

## How to test these changes

* ```...```

## Pull Request checklists

This PR is a:
- [ ] bug-fix
- [ ] new feature
- [ ] maintenance

About this PR:
- [ ] it includes tests.
- [ ] the tests are executed on CI.
- [ ] the tests generate log file(s) (path).
- [ ] pre-commit hooks were executed locally.
- [ ] this PR requires a project documentation update.

Author's checklist:
- [ ] I have reviewed the changes and it contains no misspelling.
- [ ] The code is well commented, especially in the parts that contain more complexity.
- [ ] New and old tests passed locally.

## Reviewer's checklist

Copy and paste this template for your review's note:

```
## Reviewer's Checklist

- [ ] I managed to reproduce the problem locally from the `main` branch
- [ ] I managed to test the new changes locally
- [ ] I confirm that the issues mentioned were fixed/resolved .
```

### CODE REVIEW

**Comment by xmnlab** [comment]
@Anavelyz @ayeankit @YurelyCamacho if you have time, could you review this PR please?
thanks!

File: src/scicookie/logs.py [hunk 1]
```diff
@@ -13,15 +16,20 @@ class SciCookieErrorType(Enum):
...
         os._exit(message_type.value)
 
     @staticmethod
     def info(message: str):
+        """Print an info message."""
         print(Fore.BLUE, message, Fore.RESET)
 
     @staticmethod
     def warning(message: str):
+        """Print an warning message."""
```
Comment (Anavelyz) [inline src/scicookie/logs.py:34]:
```suggestion
        """Print a warning message."""
```

File: src/scicookie/{{cookiecutter.project_slug}}/.bandit
Comment (Anavelyz) [inline src/scicookie/{{cookiecutter.project_slug}}/.bandit:1]: @xmnlab don't forget remove it https://github.com/osl-incubator/scicookie/blob/eb6f64dc9011517c099a9fbfe44d3801493186a8/src/scicookie/hooks/post_gen_project.py#L95

Comment (xmnlab) [reply src/scicookie/{{cookiecutter.project_slug}}/.bandit:1]: good catch ! thanks!

**Comment by YurelyCamacho** [comment]
@xmnlab I don't see anything out of the ordinary, though I'm still trying to figure it all out. Thank you

**Pull request merged**